projects/packages/search/src/dashboard/components/record-meter/lib/record-info.js

```diff
@@ -55,12 +55,13 @@ export default function getRecordInfo( post_count, post_type_breakdown, tier, la
 	if ( numItems > 0 && hasValidData && hasBeenIndexed ) {
 		for ( let i = 0; i < numItems; i++ ) {
 			const theData = Object.values( post_type_breakdown )[ i ];
-			const name = Object.keys( post_type_breakdown )[ i ];
+			const count = theData.count;
```

Comment:
You could use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) here if you like:

```
const { count, title: name } = theData;
```